tests/detect-itype/test.yaml

```diff
@@ -0,0 +1,27 @@
+requires:
+  min-version: 7
```

Comment:
Is this needed here? Do these rules not work on Suricata version < 7?
You can fetch suricata-6.0.x and see if the tests pass on that too on your local system. 😉

Reply:
I added this based on your comment on the other suricata-verify PR so I thought maybe its  must for all s-v PRs now.

Reply:
Should I just remove it altogather?

Reply:
Have you tried running these tests on suricata-6.0.x, to see if they pass? If they do, then you can remove it, yes :)

Reply:
How can I build an older version to test?

Reply:
There's no need to build an older version for this, it's actually about checking out a branch with https://github.com/OISF/suricata/tree/master-6.0.x and running your suricata-verify from it :)

(But if you wanted to have a test for 6, only, for instance, you could have:)
```yaml
requires:
    version: 6
```

Reply:
got it, thanks. I will try and then update

Reply:
tests pass on version 6.0.x so I am removing the min version line